src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml

```diff
@@ -38,7 +38,7 @@
                         and the settings for this project need to be migrated to the new format.
                     </Run>
                     <LineBreak />
-                    <Run>See </Run> <Hyperlink> here</Hyperlink> <Run> for more information.</Run>
+                    <Run>See </Run> <Hyperlink NavigateUri="http://any" RequestNavigate="OnNavigateToHelp"> here</Hyperlink> <Run> for more information.</Run>
```

Comment:
We have to supply a URL in the XAML otherwise the `OnNavigateToHelp` method won't be called. However, we can't easily reference the `MigrationString.LearnMoreUrl` value from the resource file.

So, we can just use a dummy URL. All the wizard does is forward the event on to the controller to handle so it doesn't actually need the URL.